src/components/CharactersFilters.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <SearchInput
+    :type="'input'"
+    :placeholder="'Name'"
+    :ariaLabel="'Search for character name'"
+    :selectedValue="searchParams.name"
+    @filterChange="(value) => setSearchParams({ name: value })"
+  />
+
+  <SearchInput
+    :placeholder="'Status'"
+    :ariaLabel="'Search for character status'"
+    :items="statusItems"
+    :selectedValue="searchParams.status"
+    @filterChange="(value) => setSearchParams({ status: value })"
+  />
+
+  <SearchInput
+    :type="'input'"
+    :placeholder="'Species'"
+    :ariaLabel="'Search for character specie'"
+    :selectedValue="searchParams.species"
+    @filterChange="(value) => setSearchParams({ species: value })"
+  />
+
+  <SearchInput
+    :type="'input'"
+    :placeholder="'Type'"
+    :ariaLabel="'Search for character type'"
+    :selectedValue="searchParams.type"
+    @filterChange="(value) => setSearchParams({ type: value })"
+  />
+
+  <SearchInput
```

Comment:
let's store params somewhere, make adjustments and use v-for. Code is almost fully repeated